src/Driver.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PodPoint\MonologKinesis;
+
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Support\Arr;
+use Monolog\Logger;
+use PodPoint\MonologKinesis\Contracts\Client;
+
+class Driver
+{
+    /**
+     * Create a new instance of a Kinesis logging driver.
+     *
+     * @param  Container  $app
+     * @param  array  $config
+     * @return Logger
+     *
+     * @throws \Illuminate\Contracts\Container\BindingResolutionException
+     */
+    public function __invoke(Container $app, array $config): Logger
+    {
+        $client = $app->make(Client::class)->configure($config);
```

Comment:
`configure` is not a method on the `Client` contract

Reply:
Well spotted <3